tracing.go

```diff
@@ -459,6 +481,18 @@ var (
 	zeroSpanID  SpanID
 )
 
+// Contains information about how the name of the transaction was determined.
+type Source string
```

Comment:
m: I would like to call this `TransactionSource` (and rename the enum below as well) to make it super clear to the reader - `Source` is kind of ambiguous.

Reply:
https://github.com/getsentry/sentry-go/pull/492/commits/204e87b8bbb7c448b266ab7987d1d370ebdbeb4b